cpp/lib/src/LocalDataDB.cc

```diff
@@ -47,11 +47,10 @@ static void CreateTables(DbConnection * const db_connection) {
 LocalDataDB::LocalDataDB(const OpenMode open_mode): single_transaction_(open_mode == READ_WRITE) {
     db_connection_ = new DbConnection(UBTools::GetTuelibPath() + "local_data.sq3" /* must be the same path as in fetch_marc_updates.py */,
                                       (open_mode == READ_WRITE) ? DbConnection::CREATE : DbConnection::READONLY);
+    CreateTables(db_connection_);
```

Comment:
Is this technically possible, even if the sq3 file has been opened in READONLY mode?